designer/apps/api/v1/serializers.py

```diff
@@ -168,6 +168,7 @@ class Meta:
         model = IndexPage
```

Comment:
I think we actually will want `model = IndexPage` to be `model = EnterprisePage`, following the convention in the `ProgramPageSerializer` where the `model` is `ProgramPage`.